components/common/FormFields.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ChangeEventHandler } from "react"
+import {
+    Button,
+    Checkbox,
+    Flex,
+    FormControl,
+    FormErrorMessage,
+    FormLabel,
+    Input,
+} from "@chakra-ui/react"
+import styled from "@emotion/styled"
+import ErrorAlert from "./ErrorAlert"
+
+type FormInputFieldProps = {
+    label: string
+    placeholder: string
+    isRequired?: boolean
+    isInvalid?: boolean
+    formErrorMessage?: string
+    onChange: ChangeEventHandler<HTMLInputElement>
+}
+
+export function FormInputField({
```

Comment:
This could be an arrow function
```typescript
export const FormInputField = ({
    label,
    placeholder,
    isRequired,
    isInvalid,
    formErrorMessage,
    onChange,
}: FormInputFieldProps): JSX.Element => (
    <FormControl isInvalid={isInvalid} isRequired={isRequired}>
        <FormLabel>{label}</FormLabel>
        <Input placeholder={placeholder} onChange={onChange} />
        {formErrorMessage && (
            <FormErrorMessage>{formErrorMessage}</FormErrorMessage>
        )}
    </FormControl>
)
```